### ISSUE TITLE
so high CPU load rate，while GPU load normal

### ISSUE THREAD

**Issue by elesun2018** [xuannianz/EfficientDet#53]
thanks
python3 train.py --snapshot imagenet --phi 0 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --batch-size 8 --steps 1000 pascal datasets/VOC2012
when i run this command on tensorflow-gpu 1.15，I have a strange problem
![image](https://user-images.githubusercontent.com/37207093/71504182-66f7e300-28b3-11ea-8a6e-f059d0394db6.png)
![image](https://user-images.githubusercontent.com/37207093/71504189-6e1ef100-28b3-11ea-9fa0-ceae41d0b9cc.png)

why so high CPU load rate，while GPU load normal

**Comment by huangeason26** [comment]
i had the same issue. so have you solve this problem yet?

**Comment by qixiuai** [comment]
How about trying to increase batch size?

**Comment by shivareddy37** [comment]
did you guys find a solution for this ??
